src/databricks/labs/blueprint/wheels.py

```diff
@@ -232,6 +232,26 @@ def upload_to_wsfs(self) -> str:
             self._installation.save(Version(self._product_info.version(), remote_wheel, self._now_iso()))
             return remote_wheel
 
+    def upload_wheel_dependencies(self, prefixes: list[str] | None = None) -> list[str]:
```

Comment:
```suggestion
    def upload_wheel_dependencies(self, prefixes: list[str]) -> list[str]:
```
make prefixes mandatory.